apps/src/utils.js

```diff
@@ -289,6 +289,7 @@ export function getPegasusHost() {
       var name = window.location.hostname.split('.')[0];
       switch (name) {
         case 'localhost':
+        case 'localhost-studio':
```

Comment:
nope - this helper function is to retrieve the pegasus url, whether you are viewing dashboard on http://localhost:3000/ or http://localhost-studio.code.org:3000/